lambdaclass/game_backend
# Start match with selected character

Closes [#1000](https://github.com/lambdaclass/curse_of_myrra/issues/1000) 

Works in conjunction with [this client PR](https://github.com/lambdaclass/curse_of_myrra/pull/1221). Both need to be merged together.

This pr is waiting for this [PR](https://github.com/lambdaclass/curse_of_mirra/pull/1221) to be unlocked 

## Motivation

Users must be able to select the character they want to play with.

## How has this been tested?

### To test it is needed to enable one or more characters in the client, since in [this PR](https://github.com/lambdaclass/curse_of_myrra/pull/1221) Muflus is the only one available. This is done by selecting the scripteable object of the character you want to enable and check the "Enabled" option. 

- To test character selection:
  - Go to character list scene, choose one and select it in the character info scene.
  - You will be taken to the main scene and the character model to appear in the middle of the screen should be the one you chose.
  - Also the `selected_character` column in your database user should be the character chosen.
  - When you play a game the character you are assigned should be the one you have selected.
